requirements.txt

```diff
@@ -2,4 +2,5 @@ scipy>=0.11
 numpy>=1.3
 pandas
 libpysal
```

Comment:
Since libpysal is a dep for esda, is it redundant to list libpysal here with esda?

Reply:
Let's try without directly specifying `libpysal` as a requirement. I will make the change and push up another commit.

Reply:
Along that same logic, do we need to specify `numpy` since `scipy` is built on top of it?